### DISCUSSION TITLE
Noise on line art

### DISCUSSION THREAD

**Discussion by Airioo** [bnpr/Malt#509] [Q&A]
Hi ! I'm wondering if it's possible in the 1.0 to do something like at this post #305 ?
Thanks !

**Comment by pragma37** [comment]
I somehow missed this question, sorry!

The nodes changed a bit but the logic should be the same.
If you go node by node searching their names you should easily find their current versions.

**Reply by Airioo** [reply]
Okay thanks I will try it :)
